bin/node-template/node/src/service.rs

```diff
@@ -100,6 +101,8 @@ pub fn new_partial(
 
 	let (grandpa_block_import, grandpa_link) = sc_consensus_grandpa::block_import(
 		client.clone(),
+		// false positive
+		#[allow(clippy::redundant_clone)]
 		&(client.clone() as Arc<_>),
```

Comment:
```suggestion
		&client,
```

This is what I meant with "should be solved properly". This should make clippy stop complaining.

Reply:
Ah, I think I tried `&(client as Arc<_>),` which did not work, after which I assumed it was a false positive. Did you cargo check that suggestion?

Reply:
Yes